5.5/examples/replica/persistent_nfs_volume.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
i understand, but i still don't think the sample should be responsible for that.  persistent volumes are a general openshift concept, not something specific to this sample, so i don't want to duplicate content related to them.